src/components/AttentionBox/__stories__/attentionBox.stories.mdx

```diff
@@ -0,0 +1,162 @@
+
+<!--- Component documentation -->
+
+# Attention Box
+- [Overview](#overview)
+- [Props](#props)
+- [Usage](#usage)
+- [Variants](#variants)
+- [Do’s and don’ts](#dos-and-donts)
+- [Use cases and examples](#use-cases-and-examples)
+- [Related components](#related-components)
+- [Feedback](#feedback)
+
+## Overview
+Attention box lets users know important information within content areas, as close as possible to the content it’s about.
+<Canvas>
+  <Story name="Overview"
+         args={{ title: "Attention box title", text: "Studies show that 100% of people who celebrate birthdays, will eventually die.", componentClassName: "monday-style-attention-box_box" }}>
+    { attentionBoxTemplate.bind({}) }
+  </Story>
+</Canvas>
+
+## Props
+<ArgsTable of={ AttentionBox } />
+
+## Usage
+<UsageGuidelines guidelines={[
+  "Use attention box if there is crucial information for user to finish or be acknowledged about a task.",
+  "Attention box do not dismiss automatically. They persist on the page until the user dismisses them or takes action that resolves the notification.",
+  "The width of attention box is based on content and layout. They can expand to the fill the container or content area they relate to.",
+  "Their height is based on the content length, which should not exceed two lines of text.",
+]}/>
+
+<Tip title="Check yourself">If the information about a component isn’t critical for the user, use a <Link href="/?path=/docs/components-tooltip--overview" size={Link.sizes.SMALL} withoutSpacing>Tooltip</Link> instead.</Tip>
+
+## Variants
+### States
+There are 4 types of attention boxes: Primary, success, danger, and dark (natural).
+<Canvas>
+  <Story name="States">
+    <MultipleStoryElementsWrapper className="monday-style-attention-box_column-wrapper">
+      <AttentionBox title="Primary" text="Lorem Ipsum is simply dummy text of the printing and typesetting industry." componentClassName="monday-style-attention-box_box" />
+      <AttentionBox title="Success" text="Lorem Ipsum is simply dummy text of the printing and typesetting industry." type={ATTENTION_BOX_TYPES.SUCCESS} componentClassName="monday-style-attention-box_box" />
+      <AttentionBox title="Danger" text="Lorem Ipsum is simply dummy text of the printing and typesetting industry." type={ATTENTION_BOX_TYPES.DANGER} componentClassName="monday-style-attention-box_box" />
+      <AttentionBox title="Dark" text="Lorem Ipsum is simply dummy text of the printing and typesetting industry." type={ATTENTION_BOX_TYPES.DARK} componentClassName="monday-style-attention-box_box" />
+    </MultipleStoryElementsWrapper>
+</Story>
+</Canvas>
+
+### Dismissable 
+<Canvas>
+  <Story name="Dismissable ">
+    <div className="monday-style-attention-box_row-wrapper">
+      <AttentionBox title="Regular attention box" text="Dismissable attention box with two lines of content." onClose componentClassName="monday-style-attention-box_box" />
```

Comment:
componentClassName is deprecated for this component, please use className instead.

Reply:
Please add mock function for onClose instead of the shorthand syntax because we except developer to pass a function value to this component and not boolean true value.
For doing this you will need to create the mock function with useCallback and write the story using anonymous function syntax like this:
`
<Canvas>
  <Story name="Dismissable ">
  {() => {
  const mockOnClose = useCallback(() =>, []);
  return <AttentionBox onClose={mockOnClose} />
  }}
 </Story>
</Canvas>
`

You can look on button success state for reference.